cmake/BuildYARP.cmake

```diff
@@ -5,10 +5,6 @@
 include(YCMEPHelper)
 include(FindOrBuildPackage)
 
-find_package(ACE QUIET)
```

Comment:
Out of interest, doesn't that break non-conda builds?

Reply:
`ACE`  needs to be installed in the system on all platforms supported by the robotology-superbuild. In general, in the superbuild build files there were a few of these `find_package`  for dependencies that are not handled by the superbuild, but to avoid problems in the conda recipe generation process I just removed them for now, so that eventually we can add them back consistently for all packages.